modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -83,6 +83,17 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 				DDMFormInstance formInstance = ddmFormDisplayContext.getFormInstance();
 				%>
 
+				<c:if test="<%= !ddmFormDisplayContext.hasValidStorageType(formInstance) %>">
+					<div class="ddm-form-basic-info">
+						<clay:container-fluid>
+							<clay:alert
+								displayType="danger"
+								message='<%= LanguageUtil.format(request, "this-form-was-created-using-a-storage-type-x-that-is-not-available-for-this-liferay-dxp-installation.-install-x-to-make-it-available-for-editing", formInstance.getStorageType()) %>'
+							/>
+						</clay:container-fluid>
+					</div>
+				</c:if>
```

Comment:
@marcosapmf, please move this alert message to be near of the other existent messages. Move this code to be just after line 165 would be great. 😄